packages/@storefront/flux-capacitor/src/flux-capacitor.ts

```diff
@@ -20,6 +20,23 @@ declare module 'redux' {
   }
 }
 
+// NOTE: `any` temporarily added to suppress compiler warnings re: `void`-type.
+// TODO: Remove `any` from union type definition.
+export type Parse = <T>(url: string) => Promise<any | { route: string, request: T }>;
+export type Build = <T>(type: string, state: Store.State) => string;
+
+// TEMP: This was originally used/expoerted by the storefront-core's UrlService.
```

Comment:
Mark this as deprecated. You might be able to use the tsdoc `@deprecated` directive.